realm/src/androidTest/java/io/realm/RealmListTest.java

```diff
@@ -53,7 +53,8 @@ protected void setUp() throws Exception {
     }
 
     // Check that all methods work correctly on a empty RealmList
-    private void testMethodsOnEmptyList(RealmList<Dog> list) {
+    private void testMethodsOnEmptyList(Realm realm, RealmList<Dog> list) {
```

Comment:
This method name should not begin with "test". The instrumentator might be smart enough to recognize it's not an actual test, but it makes it a bit harder for humans IMO